modules/ce_etherpad/src/Plugin/CollaborativeNetwork/EtherpadConnect.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Drupal\ce_etherpad\Plugin\CollaborativeNetwork;
+
+use Drupal\ce_editors\CollaborativeNetworkBase;
+use EtherpadLite\Client;
+use GuzzleHttp\Exception\ClientException;
+use GuzzleHttp\Exception\ConnectException;
+
+/**
+ * Contains Plugin ID and Plugin definition info for ce_etherpad.
+ *
+ * @CollaborativeNetwork(
+ *  id = "ce_etherpad",
+ *  label = @Translation("Etherpad"),
+ *  description = @Translation("Plugin provided by ce_etherpad."),
+ * )
+ */
+class EtherpadConnect extends CollaborativeNetworkBase implements EtherpadConnectInterface {
```

Comment:
Why `EtherpadConnect`? Any specific reason?
EtherpadEditor will be better I think.

Reply:
Not any specific reason
Renamed the file ...